jekyll/_docs/build-image-trusty.md

```diff
@@ -3,11 +3,13 @@ layout: classic-docs
 title: Ubuntu 14.04 (Trusty)
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (default is Ubuntu 12.04). You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+The default image is Ubuntu 14.04 (Trusty), but you can always change the image directly by going to 'Project Settings' -> 'Build Environment'.
 
-Please note that you need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding) to apply the new setting.
+Please note that you need to trigger a build by pushing commits to GitHub or 
```

Comment:
"To apply the new setting, you will need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding)."